code/src/OSFramework/Grid/IGrid.ts

```diff
@@ -29,7 +29,7 @@ namespace OSFramework.Grid {
         ): void;
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         changeProperty(propertyName: string, propertyValue: any): void;
-        clearAllChanges(): void;
+        clearAllChanges(clearInvalids: boolean): void;
```

Comment:
I think a better name could be, _forceClearDirtyMarks_ what do you think?

Reply:
We refer these as ValidationMarks but they are at the same time DirtyMarks.
I think changing the name to  forceClearValidationMarks, probably will be better.